### PR TITLE
update secret key in kodiak secret

### PR DESCRIPTION
**Pull request by LeoQuote**
turns out there's a bit missing change 🤦‍

### CODE REVIEW

**Review by etiennetremel — APPROVED** [review]

File: charts/kodiak/templates/secret.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ data:
   github-v3-api-root: {{ .Values.githubV3ApiRoot | b64enc | quote }}
   {{- end }}
   {{- if .Values.githubV4ApiRoot }}
```
Comment (etiennetremel) [inline charts/kodiak/templates/secret.yaml]:
```suggestion
  {{- if .Values.githubV4ApiUrl }}
```

**Comment by etiennetremel** [comment]
@LeoQuote could you also rename the key in the values.yaml file?

Use:
```
githubV4ApiUrl:
```

Instead of:
```
githubV4ApiRoot:
```

**Comment by LeoQuote** [comment]
I've been thinking about it , why this is so hard to achive such simple thing. I think there could be a better way, 

like we can just provide an "env" value , and render it as is in deployment.

you can write :

```
env:
    - name: PRIVATE_KEY
      valueFrom:
        secretKeyRef:
          name: kodiak
          key: PRIVATE_KEY
```

could this be a better solution?

**Comment by LeoQuote** [comment]
yes change in values.yaml was coverd by another pr: https://github.com/FikaWorks/helm-charts/pull/18

**Review by etiennetremel — APPROVED** [review]

**Comment by etiennetremel** [comment]
> I've been thinking about it , why this is so hard to achive such simple thing. I think there could be a better way,
> 
> like we can just provide an "env" value , and render it as is in deployment.
> 
> you can write :
> 
> ```
> env:
>     - name: PRIVATE_KEY
>       valueFrom:
>         secretKeyRef:
>           name: kodiak
>           key: PRIVATE_KEY
> ```
> 
> could this be a better solution?

Yes indeed, we can keep the current config but still add a `additionalEnv:` field in the `values.yaml` which would append custom block under `env:`.

**Pull request merged**